src/__tests__/setup.ts

```diff
@@ -46,8 +46,11 @@ sampleFeed.addItem({
     {
       name: "Joe Smith",
       email: "joesmith@example.com",
-      link: "https://example.com/joesmith",
+      link: "https://example.com/joesmith"
```

Comment:
@decebal Comma should be there, make sure you add it as attribute in object should have one (like you added one on line 52).